projects/dev.yaml

```diff
@@ -0,0 +1,104 @@
+# RESTBase config for small wiki installs
+#
+# - sqlite backend
+# - parsoid at http://localhost:8142
+# - wiki at http://localhost/w/api.php
+# 
+# Quick setup:
+# - npm install
+#   If you see errors about sqlite, you might have to `apt-get install
+#   libsqlite3-dev`.
+# - cp config.example.yaml config.yaml
+# - double-check and possibly modify lines marked with XXX, then start restbase with
+#
+#   node server
+#
+# - If all went well, http://localhost:7231/localhost/v1/page/html/Main_Page
+# should show your wiki's [[Main Page]].
+
+# First, we define some project templates. These are referenced / shared
+# between domains in the root_spec further down.
```

Comment:
This whole intro looks like a copy-paste from a wrong location.
- It's not SQLite or Cassandra - whatever you pass in the options will be used
- It's not for small wiki installs
- It's not for config.example.yaml, but for config.dev.yaml or smth